cmd/influxd/launcher/launcher_test.go

```diff
@@ -18,11 +18,9 @@ import (
 var ctx = context.Background()
 
 func TestLauncher_Setup(t *testing.T) {
-	l := launcher.NewTestLauncher(nil)
-	if err := l.Run(ctx); err != nil {
-		t.Fatal(err)
-	}
-	defer l.Shutdown(ctx)
+	l := launcher.NewTestLauncher()
+	l.RunOrFail(t, ctx)
+	defer l.ShutdownOrFail(t, ctx)
```

Comment:
It's not code you touched, but should Launcher.Shutdown() return an error, so that ShutdownOrFail() checks for the actual Launcher shutdown, not just the file deletion afterwards, and fails the test if the Launcher didn't shutdown?